src/pages/help-centre/_utility.js

```diff
@@ -1,14 +1,14 @@
-export const convertToHash = (category, label) => {
+export const convertToHash = ({ category: category }, { label: label }) => {
```

Comment:
Are these `types` for `ts` file or it's a valid `js`?

Reply:
since this is a utility and we all agreed the we will do all utility refactor after finishing the refactor on all of the pages first. We can skip typing this. Please remove the types here if it's a `.js` file

Reply:
changed it